src/file.rs

```diff
@@ -28,8 +33,13 @@ use std::time::{Duration, Instant};
 pub struct FileLoggerBuilder {
     common: BuilderCommon,
     format: Format,
+    source_location: SourceLocation,
```

Comment:
The above `common: BuilderCommon` field contains `source_location`, `level`, `channel_size`, and `kvfilterparameters`. So it seems that we don't need to re-introduce those fields here.